File: python_programming/practices/financial_calculator.py
# EB 6th Financial Calculator

income = float(input("what is your monthly income?  "))
rent = float(input("what is your monthly rent/mortgage?  "))
utilities=float(input("what is your monthy utilities?  "))
groceries = float(input("what is your monthly groceries?  "))
transportation = float(input("what is your monthly transportation?  "))

total = rent + utilities + transportation + groceries

utilities_percentage = utilities*100/income
rent_percentage = rent * 100/income
groceries_percentage = groceries*100/income
transportation_percentage = transportation*100/income

print(f"Your total monthly expenses are: ${total:,.2f}")



# # how do format to currency For an integer
# amount_int = 2,150.00

# print(f'${amount_int:,.2f}')

# # how do format to currency For a float
# amount_float = 98765.4321
# print(f'${amount_float:,.2f}')

# stuff to learn: how do convert strings into intergers and floats, how to format numbers into US currency format, how do concatenate, f strings.

savings = income * 0.10

remaining_money = income - (rent+utilities+groceries+transportation+savings)

print(f"\nYour rent is ${rent:.2f} and that is {rent_percentage:.1f}% of your income.")
print(f"Your utilities are ${utilities:.2f} and that is {utilities_percentage:.1f}% of your income.")
print(f"Your groceries are ${groceries:.2f} and that is {groceries_percentage:.1f}% of your income.")
print(f"Your transportation is ${transportation:.2f} and that is {transportation_percentage:.1f}% of your income.")
print(f"\nYou should save ${savings:.2f} a month, that is 10% of your income.")
print(f"You have ${remaining_money:.2f} of spending money each month!")

File: C_programming/notes/practices/financial_calculator.c
// EB 6th Financial Calculator

#include <stdio.h>
int main() {

    float income, rent, utilities, groceries, transportation;
    float rent_percent, utilities_percent, groceries_percent, transportation_percent;
    float savings, spending_money;
    
    printf("What is your monthly income: ");
    scanf("%f", &income);
    printf("What is your monthly rent/mortgage: ");
    scanf("%f", &rent);
    printf("What is your monthly utilities: ");
    scanf("%f", &utilities);
    printf("What is your monthly groceries: ");
    scanf("%f", &groceries);
    printf("What is your monthly transportation: ");
    scanf("%f", &transportation);

    rent_percent = (rent / income) * 100;
    utilities_percent = (utilities / income) * 100;
    groceries_percent = (groceries / income) * 100;
    transportation_percent = (transportation / income) * 100;

    savings = income * 0.10;
    spending_money = income - (rent + utilities + groceries + transportation + savings);
    printf("\nYour rent is $%.2f and that is %.0f%% of your income.\n", rent, rent_percent);
    printf("Your utilities are $%.2f and that is %.0f%% of your income.\n", utilities, utilities_percent);
    printf("Your groceries are $%.2f and that is %.0f%% of your income.\n", groceries, groceries_percent);
    printf("Your transportation is $%.2f and that is %.0f%% of your income.\n", transportation, transportation_percent);
    printf("\nYou should save $%.2f a month, that is 10%% of your income.\n", savings);
    printf("\nYou have $%.2f of spending money each month!\n", spending_money);

    
     return 0;
    
}